[docs] Improve entry points for issue repros (#19501)
mui-org/material-ui: .github/ISSUE_TEMPLATE/1.bug.md

  Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
  Include code to reproduce, if relevant (which it most likely is).

  You should use the official codesandbox template as a starting point: https://material-ui.com/r/issue-template

  If you have an issue concerning TypeScript please start from this TypeScript playground: https://material-ui.com/r/ts-issue-template

>   Issues without some form of live example have a longer response time.

Reviewer: 😆 
```suggestion
  Issues without some form of live example will be ignored and closed.
```

Author: If I'm moody, yes ;) I would introduce a "missing reproduction" label and close these if they have that label for 7 days.

Reviewer: What about the `incomplete` label, and close with this message?

The issue has been closed because it does not conform to our issue requirements. 
Please provide a full reproduction test case. This would help a lot 👷 .
A live example would be perfect. [This **codesandbox.io** template](https://codesandbox.io/s/github/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript) _may_ be a good starting point. Thank you!

Author: Yep and then use the issue template instead of the example.
